From f08a22526c050d2032c926c5bc045927a646ea4a Mon Sep 17 00:00:00 2001
From: Kalev Lember <kalevlember@gmail.com>
Date: Wed, 16 Oct 2013 12:01:36 +0200
Subject: [PATCH] background: Fix test-chooser-dialog build with some linkers

List missing libs for the libbackground-chooser.la convenience library.

https://bugzilla.gnome.org/show_bug.cgi?id=709302
---
 panels/background/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/panels/background/Makefile.am b/panels/background/Makefile.am
index f24f8b3..2766207 100644
--- a/panels/background/Makefile.am
+++ b/panels/background/Makefile.am
@@ -40,6 +40,8 @@ libbackground_chooser_la_SOURCES =	\
 	bg-colors-source.c		\
 	bg-colors-source.h
 
+libbackground_chooser_la_LIBADD = $(PANEL_LIBS) $(BACKGROUND_PANEL_LIBS)
+
 libbackground_la_SOURCES =		\
 	cc-background-panel.c		\
 	cc-background-panel.h
-- 
1.8.4.rc3

